### PR TITLE
Add command line usage information

### PR DESCRIPTION
**Pull request by JayHoltslander**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,7 +136,9 @@ npm link .
 
 ## Usage
 
-For command line usage please see `html-minifier --help`
+Note that almost all options are disabled by default. For command line usage please see `html-minifier --help` for a list of available options. Experiment and find what works best for you and your project.
+
+* **Sample command line:** ``html-minifier --collapse-whitespace --remove-comments --remove-optional-tags --remove-redundant-attributes --remove-script-type-attributes --remove-tag-whitespace --use-short-doctype --minify-css true --minify-js true``
```
Comment (alexlamsl) [inline README.md:141]: `--minify-css --minify-js` would do, i.e. lose those `true`s as they are the defaults.

**Review by alexlamsl — CHANGES_REQUESTED** [review]
Thanks for your contribution!

Just one minor nit & we will be good to go.

File: README.md (hunk 1 above)
Comment (DamonHD) [reply README.md:141]: It wasn't at all obvious to me how to supply (non-default / any) arguments, so I have have never used any with the switches so far, so maybe let this stand to give an extra clue to slow people such as me?

Comment (alexlamsl) [reply README.md:141]: Sure thing.

**Review by alexlamsl — APPROVED** [review]

**Pull request merged**